### PR TITLE
added stream to file type icons

### PR DESCRIPTION
**Pull request by azra-bandukwala**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11375)

### CODE REVIEW

**Comment by Jahnp** [comment]
@bigbadcapers FYI, I think you're a little closer to this code than I am—can you take a look when you have a chance?

File: packages/file-type-icons/src/FileIconType.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export enum FileIconType {
   genericFile = 3,
   listItem = 4,
   sharedFolder = 5,
-  multiple = 6
+  multiple = 6,
+  stream=7
```
Comment (ecraig12345) [inline packages/file-type-icons/src/FileIconType.ts]: Minor formatting issue. This should have been corrected automatically by Prettier when you committed, which makes me think your local copy isn't quite set up properly. 
```suggestion
  stream = 7
```

File: change/@uifabric-file-type-icons-2019-12-04-16-34-43-azbanduk-add-stream-icon.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "ComboBox: empty errorMessage shouldn't create white space. (#11353)",
```
Comment (ecraig12345) [inline change/@uifabric-file-type-icons-2019-12-04-16-34-43-azbanduk-add-stream-icon.json]: Can you update this message to describe what the change does?

Comment (azra-bandukwala) [reply change/@uifabric-file-type-icons-2019-12-04-16-34-43-azbanduk-add-stream-icon.json]: Done!

**Pull request merged**